frictionless/helpers.py

```diff
@@ -4,6 +4,7 @@
 import csv
 import json
 import glob
+import jinja2
```

Comment:
I think `jinja2` is not in our `INSTALL_REQUIRES`. We need to add it otherwise it will fail on the user's machine (CI can't catch it)